README.md

```diff
@@ -43,9 +43,21 @@ out, running through [some tutorials](https://google.github.io/xls/tutorials/),
 [reporting bugs](https://github.com/google/xls/issues), and letting us know
 what you think!
 
+## Install using conda
+
+```console
+curl -O https://repo.anaconda.com/miniconda/Miniconda3-py37_23.1.0-1-Linux-x86_64.sh
```

Comment:
@cdleary do you want shell prompt and output in there?

I personally prefer it that way so that you can easily cut and paste.

Reply:
Go for it!